### PR TITLE
Finalize 1.20 for ViaFabric. (sorta)

### PR DESCRIPTION
**Pull request by Kichura**
Updates the minecraft dependency from release candidate 1 -> final,
Mod menu isn't updated yet although can be looked into a third time in case it is,
Publishing of 1.20 is stated in curseforge and should work out hopefully this time.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 yaml_version=2.0
 
-publish_mc_versions=1.19.4, 1.18.2, 1.17.1, 1.16.5, 1.15.2, 1.14.4, 1.8.9
-# example: 1.19.1-rc2
-modrinth_mc_snapshot=1.20-rc1
-# example: 1.19-Snapshot
-curseforge_mc_snapshot=1.20-Snapshot
+publish_mc_versions=1.20, 1.19.4, 1.18.2, 1.17.1, 1.16.5, 1.15.2, 1.14.4, 1.8.9
+# example: 1.19.1
+modrinth_mc_snapshot=1.20
```
Comment (creeper123123321) [inline gradle.properties:11]: Just make it blank

**Review by creeper123123321 — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (Kichura) [reply gradle.properties:11]: on it.

Comment (Kichura) [reply gradle.properties:11]: Removed it but github isn't displaying the change for some reason.

**Comment by Kichura** [comment]
Mod menu is out for 1.20, went ahead and updated that one now.

**Review by creeper123123321 — APPROVED** [review]

**Pull request merged**